### PR TITLE
Fix product width on featuredproducts

### LINKED ISSUE #26705: ps_crossselling - FO - products display is missing spaces
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In ps_crossselling module v2.0.1 with PS 1.7.8.x, the display of the products isn't correct. There are no spaces between each product.
Screenshot 2021-11-22 at 15 43 54

With PS 1.7.7.7, it was OK :
Screenshot 2021-11-22 at 16 10 48

### Expected behavior

It should look like what we have in module featured products :
Screenshot 2021-11-22 at 15 44 15

### Steps to reproduce

1. Make sure module cross selling is installed
2. Go to BO > Edit a product > Add Related Products
3. Go to FO > Product page > See display error

### PrestaShop version(s) where the bug happened

1.7.8.1

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

ps_crossselling v2.0.1

### LINKED ISSUE #26992: FO - A bad display when we hover on the product in the featured-products list
### Prerequisites

- [X] I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
- [X] I have already [searched in existing issues](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3ABug) and found no previous report of this bug.

### Describe the bug and add screenshots

In the FO > featured-products list, when we hover on a product => there is a bad display
image

It was ok with PS1.7.8.2

### Expected behavior

It should be 
image

### Steps to reproduce

1. Go to FO > Home page
2. In the Popular product list
3. Hover on a product with the mouse
4. See an error 

### PrestaShop version(s) where the bug happened

1.7.8.x (1.7.8.3)

### PHP version(s) where the bug happened

7.3

### If your bug is related to a module, specify its name and its version

_No response_

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 1.7.8.x
| Description?      | There was a display flex on product which was causing the size to be a bit smaller before, as it's considered as a regression I reverted it
| Type?             | bug fix
| Category?         | FO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #26992.
| How to test?      | See issue
| Possible impacts? | Product list 

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26998)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]
lgtm, I restarted the CI

**Review by matks — COMMENTED** [review]
Looks OK but CI fails.
We need https://github.com/PrestaShop/PrestaShop/pull/26987 to be merged to fix CI

**Review by Progi1984 — DISMISSED** [review]

**Comment by florine2623** [comment]
Hello @NeOMakinG ,

In PS 1.7.8.x I don't have a width problem on featuredproducts : 
Screenshot 2021-12-20 at 09 44 14

But on develop, the issue is there :
Screenshot 2021-12-20 at 09 43 25

I tested on develop, it is OK ✅ . What do you think ?
Thanks!

**Comment by khouloudbelguith** [comment]
Hi @NeOMakinG,

In fact, we found another major regression very similar to this issue.
https://github.com/PrestaShop/PrestaShop/issues/27030 => it is displayed on a lot of pages.

- Brands list page: shop.com/index.php?id_manufacturer=2&controller=manufacturer&id_lang=1
- Supplier list page: shop.com/index.php?id_supplier=2&controller=supplier&id_lang=1
- Search list page: shop.com/index.php?controller=search&s=mug
- Same issue in the FO > ps_specials module
- FO > Product page > ps_crossselling module & ps_viewedproduct module
- Also, in the featured product list => if we resize the window => display NOK and it became a major issue 
image

@NeOMakinG, what do you think, this PR could be updated to fix the first issue: https://github.com/PrestaShop/PrestaShop/issues/26992 and the second issue: https://github.com/PrestaShop/PrestaShop/issues/27030?

Thank you!

**Comment by florine2623** [comment]
Hello @NeOMakinG @khouloudbelguith ,

Has been tested on :
Brand list
Supplier list
Search list
ps_specials 
ps_viewedproduct
ps_crossselling

The product width and Quickview are well displayed on screen bigger than 765px ✅ 

On screen sizes smaller than 765px, the Quickview doesn't display 🚫 :

https://user-images.githubusercontent.com/16019289/146935478-083f06bc-3d36-4ae6-8445-03bdb491f24a.mov

Also, on ps_crossselling, it should have spacing between each products, now we have :
Screenshot 2021-12-21 at 14 11 09

Thanks!

**Comment by NeOMakinG** [comment]
@khouloudbelguith @florine2623 every issues you mentioned should be fixed inside this PR now!

**Comment by florine2623** [comment]
Hello @NeOMakinG ,

Thanks for the update. It works well on my end !
Once I get approvals, I will add the QA approved ✅ label

Thanks!

**Review by matthieu-rolland — APPROVED** [review]

**Review by atomiix — APPROVED** [review]

**Comment by matthieu-rolland** [comment]
thank you @NeOMakinG !

**Pull request merged**
